x-pack/test/apm_api_integration/tests/services/annotations.spec.ts

```diff
@@ -79,52 +100,46 @@ export default function annotationApiTests({ getService }: FtrProviderContext) {
       });
 
       it('fails with a 400 bad request if data is missing', async () => {
-        const response = await request({
-          url: '/api/apm/services/opbeans-java/annotation',
-          method: 'POST',
-        });
+        const err = await expectToReject<ApmApiError>(() =>
+          // @ts-expect-error
+          createAnnotation()
+        );
 
-        expect(response.status).to.be(400);
+        expect(err.res.status).to.be(400);
       });
 
       it('fails with a 400 bad request if data is invalid', async () => {
```

Comment:
Can you make this a separate test?

```suggestion
      it('fails with a 400 bad request if timestamp is invalid', async () => {
```

And if we specifically want to test timestamp, make sure to add the other arguments.